entrypoint.sh

```
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+set -e
+
+echo "Waiting for Postgres at db:5432..."
+until pg_isready -h db -p 5432 -U cinema_user >/dev/null 2>&1; do
+  sleep 1
```

`set -e` is good for failing fast. The waiting loop is fine technically, but prefer calling the Python `wait_for_db` management command here (`python manage.py wait_for_db`) to meet the explicit project requirement and keep logic in one place (Python).